schemas/JSON/manifests/v1.5.0/manifest.defaultLocale.1.5.0.json

```diff
@@ -0,0 +1,213 @@
+{
+  "$id": "https://aka.ms/winget-manifest.defaultlocale.1.5.0.schema.json",

```

Comment:
I reallllllllllllllllly wish we had a single version file so that we could actually see the diff.

If there was a `manifests/latest` directory with unversioned schemas and you just copied the new files' contents into them, I think that would achieve the desired result.  Thoughts?

One could achieve similar results with a little more effort for the reviewer if one always started with a commit of the previous version's files, just renamed.  Meaning, the first commit of such a PR is always the result of "copy version N-1 files to version N directory with new names".  That way the reviewer can look at everything but the first commit to get the schema diff.

Reply:
It's a copy of v1.4 schemas(with only 1.4.0 replaced by 1.5.0), maybe I could make the copy and update to1.5.0 separate commits

Reply:
No, if you aren't changing it to make future diffs easy, then I think just putting that in the PR initial comment would be helpful.